test/controllers/loginController.spec.js

```diff
@@ -0,0 +1,154 @@
+let mocha = require('mocha')
+let chai = require('chai')
+let chaiHttp = require('chai-http')
+let _ = require('lodash')
+
+let app = require('../../src/app.js')
+let Sequelize = require('sequelize')
+let db = require('../../src/db.js')
+
+let should = chai.should
+let expect = chai.expect
+chai.use(chaiHttp)
+
+describe ('Login Controller', () => {
+  describe ('loginController#postLogin', () => {
+    let testUser = null
+
+    beforeEach((done) => {
+      let promises = _.concat([],
+        db.User.create({
+          email: 'tnpetresky+0@gmail.com',
+          password: 'password',
+          universityId: 0
```

Comment:
I dont think 0 can ever be a valid id as far as i know. Thought incremental primary keys start at 1, no?